pages/contact.tsx

```diff
@@ -85,8 +85,8 @@ export default function ContactPage() {
         metaContent={content.meta.content}
       />
 
-      <SectionContainer sx={{ textAlign: 'center', mt: 5 }}>
-        <Typography variant="h1" sx={{ textAlign: 'center' }}>
+      <SectionContainer sx={{ textAlign: 'left', mt: 5 }}>
```

Comment:
Might be worth adding some margins on the left here, as the text is right on the edge of the page
![currentView](https://i.imgur.com/Xgj4m16.png)

Reply:
Def yes! I am getting this margin even on 1150w screen.

Reply:
On the about page as well, thinking it looks like when the paper section is not used, the margins are off.